React/Views/RCTBorderCurve.h

```diff
@@ -0,0 +1,13 @@
+/*
+  * Copyright (c) Facebook, Inc. and its affiliates.
+  *
+  * This source code is licensed under the MIT license found in the
+  * LICENSE file in the root directory of this source tree.
+  */
+
+#import <Foundation/Foundation.h>
+
+typedef NS_ENUM(NSInteger, RCTBorderCurve) {
+	RCTBorderCurveContinuous = 0,
+	RCTBorderCurveCircular,
+};
```

Comment:
Hi, could you add a new line in this file? Our internal linter is complaining about this.

Reply:
Hi, yes of course, should be ok now